tests/unit/components/common/StatusList.spec.ts

```diff
@@ -29,4 +31,29 @@ describe('StatusList.vue', () => {
     })
     expect(wrapper.find('[data-test="select-status"]').exists()).toBeTruthy()
   })
+
+  it('removes Refund Rejected option', async () => {
+    // Mock the API call to return the list of routing slip statuses
+    const sandbox = sinon.createSandbox()
+    const get = sandbox.stub(CodesService, 'getCodes')
+    get.returns(new Promise(resolve => resolve({ data: { codes: routingSlipStatusListMock } })))
+
+    // Render the StatusList component
+    const wrapper = mount(StatusList, {
+      localVue,
+      vuetify,
+      propsData: {
+        value: 'ACTIVE'
+      }
+    })
+
+    // Wait for the component to finish rendering
+    await wrapper.vm.$nextTick()
+
+    // taking all of the descriptions
+    const descriptions = wrapper.vm.routingSlipStatusList.map(status => status.description)
+    // Check if the "Refund Rejected" option is not in the status list
+    expect(descriptions).toContain('Active')
+    expect(descriptions).not.toContain('Refund Rejected')
```

Comment:
You need to add the `Refund Rejected` code/desc to this mock for this test to be valid https://github.com/bcgov/fas-ui/blob/main/tests/unit/test-data/mock-code.ts